build/Helix/CreateHelixTestFailureDumps.ps1

```diff
@@ -40,23 +55,23 @@ foreach ($testRun in $testRuns.value)
         if (-not $workItems.Contains($workItem))
         {
             $workItems.Add($workItem)
-
-            Out-File -FilePath $outputFilePath -Append -InputObject "<h3>$helixWorkItemName</h3>"
-
             $filesQueryUri = "https://helix.dot.net/api/2019-06-17/jobs/$helixJobId/workitems/$helixWorkItemName/files"
-
             $files = Invoke-RestMethod -Uri $filesQueryUri -Method Get
 
+            $screenShots = $files | where { $_.Name.EndsWith(".jpg") }
```

Comment:
Consider linking to the console.log files also, since that is useful.

In fact, it might be good just to include all the files here. It makes sense to have them sorted into categories, but maybe have a "misc" category for everything else?

Reply:
That would mean that the file would be more "crowded" since every work item would generate a list of files. But I think that is ok. If someone is looking for something in particular, they can always ctrl+f for "screenshots" or whatever.

---
In reply to: [327317628](https://github.com/microsoft/microsoft-ui-xaml/pull/1130#discussion_r327317628) [](ancestors = 327317628)

Reply:
Added a Misc section to put everything there. I did a test run here https://dev.azure.com/ms/microsoft-ui-xaml/_build/results?buildId=38182.

I made it only dump out the files for failed tests (I don't think we really care about successful ones?) so that it doesn't feel too "crowded".

Reply:
Looks great,